### ISSUE TITLE
Landmark for Cookie banner/ Privacy banner

### ISSUE THREAD

**Issue by sareh** [bbc/simorgh#1707]
**Describe the bug**
The cookie banner and privacy banner don't have aria landmarks. They should, since they're top-level regions of the page. 

The outer div wrapping these should have a `role="region"` landmark. 

**To Reproduce**

1. Go to an article https://www.bbc.co.uk/news/articles/clldg965yzjo
2. Run the Chrome axe accessibility extension https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd 
3. See error `All page content must be contained by landmarks`

<img width="274" alt="Screen Shot landmark error" src="https://user-images.githubusercontent.com/3028997/57701489-02558400-7654-11e9-8fee-e89ce72e3552.png">

**Expected behavior**
There should be no error.

**Comment by greenc05** [comment]
@sareh might the cookie/privacy banner be better placed inside our 'banner' landmark rather than a being a standalone region? 'A banner landmark identifies site-oriented content at the beginning of each page within a website.' https://www.w3.org/TR/wai-aria-practices/examples/landmarks/banner.html
